src/main/scala/com/mongodb/spark/config/MongoCompanionConfig.scala

```diff
@@ -169,9 +190,23 @@ trait MongoCompanionConfig extends Serializable {
    */
   def create(options: util.Map[String, String], default: Self): Self
 
-  protected def prefixLessOptions(options: collection.Map[String, String]): collection.Map[String, String] =
+  /**
+   * Strip the prefix from options
+   *
+   * @param options options that may contain the prefix
+   * @return prefixLess options
+   */
+  def prefixLessOptions(options: collection.Map[String, String]): collection.Map[String, String] =
```

Comment:
Method name reads strangely, if you read "prefix" as a verb, which I did.  How about calling it stripPrefix?